### ISSUE TITLE
Add mqtt.acl.verify option to uhppoted.conf

### ISSUE THREAD

**Issue by uhppoted** [uhppoted/uhppoted-mqtt#6]
### Discussed in https://github.com/uhppoted/uhppoted/discussions/17

<div type='discussions-op-text'>

<sup>Originally posted by **irvintim** September 24, 2022</sup>
Hi:

So, my project has been moving along slowly, the next thing I want to accomplish is to be able to load up the ACLs (cards) via a Lambda job.  I'm storing the card data in DynamoDB, and then when that is updated I trigger a Lambda to grab the latest list of cards, generate a new *.acl file and tar that up and upload it to S3.  Then send an "acl:download" MQTT message with the URL for this S3 file.  Everything is working great until I get the following:

```
{
  "message": {
    "error": {
      "client-id": "test",
      "error": {
        "code": 400,
        "debug": ": no RSA public key",
        "message": "Error downloading ACL"
      },
      "method": "acl:download",
      "nonce": 6,
      "request-id": "AH1635G3",
      "server-id": "uhppoted"
    }
  },
  "hmac": "xxxx"
}
```

I found the instructions in the repo on how to create the RSA signed acl files, but I'd rather not have to do that (it's already hard enough to create a gziped tar file in a python-based Lambda job without writing anything to disk).  Is there a way to turn this "feature" off?

I am running [uhppoted-mqtt v0.8.1-beta](https://github.com/uhppoted/uhppoted-mqtt/releases/tag/v0.8.1)

Tim</div>

**Comment by uhppoted** [comment]
Ok, after some thought, this is the functionality to be implemented:

- [ ] `mqtt.acl.verify` should support multiple options - initially, _none_, _not-empty_ and _RSA_
- [ ] ACL verification should default to _RSA_ 
- [ ] ACL verification should allow both RSA and not-empty to coexist e.g. for a signed but empty ACL
- [ ] add mime-type to download request (default: _application/tar+gzip_). Initially the only supported mime-types will
      be _application/tar+gzip_ and _text/tab-separated-values_
- [ ] Parse file as TSV if message mime-type is _text/tab-separated-values_

**Comment by uhppoted** [comment]
Closing - released in [v0.8.2](https://github.com/uhppoted/uhppoted-mqtt/releases/tag/v0.8.2)
